modules/storages/spec/features/admin_storages_spec.rb

```diff
@@ -291,6 +293,9 @@
         aggregate_failures 'General information' do
           within_test_selector('storage-general-info-form') do
             fill_in 'storages_one_drive_storage_name', with: 'My OneDrive', fill_options: { clear: :backspace }
+            fill_in 'storages_one_drive_storage_tenant_id',
+                    with: '029d4741-a4be-44c6-a8e4-e4eff7b19f65',
+                    fill_options: { clear: :backspace }
```

Comment:
_nit_  we shouldn't need to clear the input if we no longer have a default storage name

```suggestion
                    with: '029d4741-a4be-44c6-a8e4-e4eff7b19f65'
```

Reply:
removed